docs/development_submitting.md

```diff
@@ -14,30 +14,32 @@ Submit your improvements, fixes, and new features to Home Assistant one at a tim
 
 2. Make your changes, create a [new platform](creating_platform_index.md), develop a [new component](creating_component_index.md), or fix [issues](https://github.com/home-assistant/home-assistant/issues).
 
-3. [Test your changes](development_testing.md) and check for style violations.
+3. [Test your changes](development_testing.md) and check for style violations.  
+    Consider adding tests to ensure that your code works.
 
 4. If everything looks good according to these [musts](development_checklist.md), commit your changes:
 
     `git add .`
 
-    `git commit -m "Add some-feature"`
+    `git commit -m "Add some feature"`
```

Comment:
I still wonder if it's best to use a non-generic example here that complies fully with the checklist below. Maybe:

```txt
git commit -m "Make lights dimmable"
```

Reply:
Sounds good to me. Add the word "example" in this case.